### DISCUSSION TITLE
CRSF V2 slow

### DISCUSSION THREAD

**Discussion by robthomson** [ExpressLRS/ExpressLRS#1385] [General]
Hello

Does express lrs support low speed crsf?

I ask because my jeti exbus to crossfire module convertor has recently had a number of people asking of expresslrs is supported.

Current setup of my module only works reliably with CRSF V2 slow

115200 baud
16 refresh rate.

https://github.com/robthomson/ExBus2CRSF

Mostly this is no issue as exbus is about the same speed.

Any ideas?

Rob

**Comment by pkendall64** [comment]
Yes ELRS can run at 115200 baud. Our lowest refresh rate on 2.4GHz is 50Hz and on 900Mhz it's 25Hz.

**Reply by robthomson** [reply]
Great stuff.

I have a module arriving tomorrow for testing.  Will shout if anything does not work!
